vpr/src/base/read_options.cpp

```diff
@@ -1754,6 +1785,97 @@ argparse::ArgumentParser create_arg_parser(std::string prog_name, t_options& arg
         .default_value("false")
         .show_in(argparse::ShowIn::HELP_ONLY);
 
+    place_grp.add_argument(args.place_static_move_prob, "--place_static_move_prob")
+        .help(
+            "The Probability of different move in Simulated Annealing."
+            "The order of moves is: {uniformMove, MedianMove, CentroidMove, WeightedCentroid, WeightedMedian, Timing feasible Region(TFR), Critical UniformMove}")
+        .nargs('+')
+        .default_value({"100", "0", "0", "0", "0", "0", "0"})
+
+        .show_in(argparse::ShowIn::HELP_ONLY);
+
+    place_grp.add_argument(args.place_static_notiming_move_prob, "--place_static_notiming_move_prob")
+        .help(
+            "The Probability of different non timing move in Simulated Annealing."
```

Comment:
typo:  "non-timing moves in Simulated Annealing placement."
How does this interact with the earlier --place_static_move_prob argument?  It seems to overlap it.  Or does it control the moves during the earlier part of the anneal (state 0) and the other argument controls the probabilities during the later part of the anneal (state 1)?  If so, you should say that.

Reply:
This option is valid for non-timing driven placement in which only non-timing directed moves are available. I have updated the help of this option to highlight that.